packages/unenv-preset/src/index.ts

```diff
@@ -1 +1 @@
-export { cloudflare } from "./preset";
+export { getCloudflarePreset } from "./preset";
```

Comment:
Do we expect anyone using our unenv-preset directly? This looks like a breaking change to me. How about this?

```suggestion
import { getCloudflarePreset } from "./preset";

export {
  cloudflare: getCloudflarePreset(),
  getCloudflarePreset,
};
```

Reply:
Good catch!